Backend/Remora.Discord.Gateway/DiscordGatewayClient.cs

```diff
@@ -677,23 +486,23 @@ private async Task<Result> CreateNewSessionAsync(CancellationToken ct = default)
             ? default
             : new Optional<IUpdatePresence>(_gatewayOptions.Presence);
 
-        SubmitCommand
+        await EnqueueCommandAsync
         (
             new Identify
             (
                 _tokenStore.Token,
                 _gatewayOptions.ConnectionProperties,
-                false,
-                _gatewayOptions.LargeThreshold,
-                shardInformation,
-                initialPresence,
-                _gatewayOptions.Intents
-            )
+                Intents: _gatewayOptions.Intents,
+                Compress: false,
+                Shard: shardInformation,
+                Presence: initialPresence
+            ),
+            ct
```

Comment:
`LargeThreshold` appears to be absent. Is your branch out of date?

Reply:
Yes, it was. I did attempt to do a merge but it was 0330 at that point. Thanks, I'll have to comb over again.